packages/backend/server/schema.prisma

```diff
@@ -252,6 +252,8 @@ model WorkspaceFeature {
 model Snapshot {
   workspaceId String   @map("workspace_id") @db.VarChar
   id          String   @default(uuid()) @map("guid") @db.VarChar
+  title       String?  @db.VarChar
+  summary     String?  @db.VarChar
```

Comment:
should add these fields to WorkspaceDoc

Reply:
Oh, no, this table data might not always be available, the snapshot is more reliable.